src/View/index.js

```diff
@@ -0,0 +1,162 @@
+      minHeight: OIOResponsiveObjectPropType,
+      maxHeight: OIOResponsiveObjectPropType,
+      minWidth: OIOResponsiveObjectPropType,
+      maxWidth: OIOResponsiveObjectPropType,
+      onClick: PropTypes.func,
+      onKeyUp: PropTypes.func,
+      onScroll: PropTypes.func,
+      order: OIOResponsiveObjectPropType,
+      overflow: OIOResponsiveObjectPropType,
+      padding: OIOResponsiveObjectPropType,
+      paddingTop: OIOResponsiveObjectPropType,
+      paddingLeft: OIOResponsiveObjectPropType,
+      paddingRight: OIOResponsiveObjectPropType,
+      paddingBottom: OIOResponsiveObjectPropType,
+      paddingHorizontal: OIOResponsiveObjectPropType,
+      paddingVertical: OIOResponsiveObjectPropType,
+      position: OIOResponsiveObjectPropType,
+      scroll: OIOResponsiveObjectPropType,
+      style: PropTypes.object,
+      textAlign: OIOResponsiveObjectPropType,
+      top: OIOResponsiveObjectPropType,
+      left: OIOResponsiveObjectPropType,
+      right: OIOResponsiveObjectPropType,
+      bottom: OIOResponsiveObjectPropType,
+      WebkitOverflowScrolling: OIOResponsiveObjectPropType,
+      width: OIOResponsiveObjectPropType,
+      zIndex: OIOResponsiveObjectPropType
+   }
+
+   static defaultProps = {
+      className: '',
+      display: r`block`,
+      onClick: () => {},
+      onKeyUp: () => {},
+      onScroll: () => {},
+      position: r`relative`,
+      style: {}
+   }
+
+   render() {
+      const {
+         children, className,
+         display, float, position, top, left, right, bottom,
+         flex, flexFlow, justifyContent, alignItems, order,
+         height, width, maxHeight, maxWidth, minHeight, minWidth,
+         margin, marginBottom, marginLeft, marginRight, marginTop,
+         padding, paddingBottom, paddingLeft, paddingRight, paddingTop,
+         border, borderBottom, borderLeft, borderRight, borderTop,
+         backgroundColor, backgroundImage, backgroundPosition, backgroundSize,
+         borderRadius, boxShadow, textAlign, zIndex,
+         overflow, WebkitOverflowScrolling,
+         onClick, onKeyUp, onScroll
+      } = this.props
+
+      // Generate CSS responsive styles with Emotion
+      /* eslint-disable object-property-newline */
+      const responsiveStyles = generateResponsiveStyles({
+         display, float, position, top, left, right, bottom,
+         flex, flexFlow, justifyContent, alignItems, order,
+         height, width, maxHeight, maxWidth, minHeight, minWidth,
+         margin, marginBottom, marginLeft, marginRight, marginTop,
+         padding, paddingBottom, paddingLeft, paddingRight, paddingTop,
+         border, borderBottom, borderLeft, borderRight, borderTop,
+         backgroundColor, backgroundImage, backgroundPosition, backgroundSize,
+         borderRadius, boxShadow, textAlign, zIndex,
+         overflow, WebkitOverflowScrolling
+      })
+      /* eslint-enable */
+
+      return (
+         <div
+            ref={this.props.forwardedRef}
```

Comment:
I'd maybe just do a unit test to verify the forwarded ref works (to prevent against unintentional future problems where a wrapper component gets in between the div and the `forwardRefToWrappedComponent` hoc